### PR TITLE
Ch95/create statistics on endpoints

### PR DESCRIPTION
**Pull request by AlineRibeiro**

### CODE REVIEW

File: app/controllers/stats_controller.rb [hunk 1]
```diff
@@ -6,6 +6,14 @@ class StatsController < ApplicationController
   def show
```
Comment (guialbuk) [inline app/controllers/stats_controller.rb:6]:
```
app/controllers/stats_controller.rb:6:3: C: Metrics/AbcSize: Assignment Branch Condition size for show is too high. [<6, 22, 1> 22.83/17]
```

This Rubocop warning means that this method is getting complex, making it harder to read and maintain.

You can extract parts of it to private methods or even create a service class for retrieving the stats data.

File: app/controllers/stats_controller.rb [hunk 2]
```diff
@@ -6,6 +6,14 @@ class StatsController < ApplicationController
...
     total_stream_time = VideoStreamEvent.where(user: current_user).sum(:duration).round
 
-    render json: { stats: { time_streamed: total_stream_time } }
+    stream_time_last_24h = VideoStreamEvent.where(user: current_user,
+                                                  created_at: 24.hours.ago..Time.zone.now).sum(:duration).round
```
Comment (guialbuk) [inline app/controllers/stats_controller.rb]:
```ruby
24.hours.ago..Time.zone.now
````

Great usage of ranges! 👏

**Review by guialbuk — COMMENTED** [review]
⛳  ... 🏌️‍♀️

File: app/controllers/stats_controller.rb (hunk 2 above)
Comment (AlineRibeiro) [reply app/controllers/stats_controller.rb]: Thank you!

**Pull request merged**